src/Features/LanguageServer/ProtocolUnitTests/Completion/CompletionTests.cs

```diff
@@ -36,7 +36,7 @@ void M()
                 locations["caret"].Single(), triggerCharacter: "\0", triggerKind: LSP.CompletionTriggerKind.Invoked);
 
             var expected = CreateCompletionItem("A", LSP.CompletionItemKind.Class, new string[] { "Class", "Internal" },
-                completionParams, commitCharacters: CompletionRules.Default.DefaultCommitCharacters);
+                completionParams);
```

Comment:
why did this chnge?  wouldn't we still expect the item to have the default commit characters?

in other words, i would expect any internal changes to the representatino to not be observable in terms of waht tests are asking/verifying.

Reply:
the json payload for the expected completion item doesn't need to include the default commit characters (instead those would come from the set specified on initialize).  since the completion item in the test is using default commit chars, they shouldn't be on the payload anymore now that we're not always sending them

Reply:
i woudl not expect us to test the payloads, but the end impact in terms of what the client perceives.  So in this case, i would still think teh client would perceive having that set of completion items through some means.

In other words, i think this should be validating expected behavior is maintained here, not the impl details of exactly what data we send.

Reply:
> In other words, i think this should be validating expected behavior is maintained here, not the impl details of exactly what data we send.

I agree, but the tests are currently implemented only to check an exact match of the payload. Outside of reworking all of the completion tests, this is correct as per how the tests currently work.

Reply:
Yeah I think that's outside the scope of this PR - I think we should validate behavior too, but that requires being able to run and setup an actual LSP client in the test framework.  Using the VS client isn't easy as we'd need to setup editor textviews/buffers to active them.  Filed https://github.com/dotnet/roslyn/issues/51101 to continue tracking this.